core/src/components/toggle/toggle.ios.vars.scss

```diff
@@ -75,3 +75,9 @@ $toggle-ios-item-end-padding-bottom:   5px !default;
 
 /// @prop - Padding start of the toggle positioned on the end in an item
 $toggle-ios-item-end-padding-start:    $item-ios-padding-start !default;
+
+/// @prop - The text color of the on/off labels
+$toggle-ios-on-off-label-color: #b0b0b0 !default;
```

Comment:
The color on light mode is not accessible:

![Screen Shot 2022-06-15 at 2 09 01 PM](https://user-images.githubusercontent.com/2721089/173895670-ca881e0e-bfe7-45dc-84a3-57aaf4fcfbad.png)

Dark mode seems fine though 

![Screen Shot 2022-06-15 at 2 10 09 PM](https://user-images.githubusercontent.com/2721089/173895814-853e293e-36b3-4e59-addd-16fe7a8070a6.png)